README.md

```diff
@@ -1,30 +1,50 @@
 [![status](https://secure.travis-ci.org/contra/gulp-beautify.png?branch=master)](https://travis-ci.org/contra/gulp-beautify)
 
 # Information
+
 <table><br><tr><br><td>Package</td><td>gulp-beautify</td><br></tr><br><tr><br><td>Description</td><br><td>Asset beautification using node-beautify</td><br></tr><br><tr><br><td>Node Version</td><br><td>>= 0.4</td><br></tr><br></table>
 
 # Usage
+
 This is a gulp plugin for js-beautify.
 
 ```javascript
 var beautify = require('gulp-beautify');
 
 gulp.task('beautify', function() {
-  gulp.src('./src/*.js')
-    .pipe(beautify({indent_size: 2}))
-    .pipe(gulp.dest('./public/'))
+  gulp
+    .src('./src/*.js')
```

Comment:
Should we add the `beautify.js(...)` example and then move this below all the other ones and mark it as the old pattern?

Reply:
That's a good idea. I will be more inline with [js-beautify documentation](https://github.com/beautify-web/js-beautify#nodejs-javascript-1)